rdflib/graph.py

```diff
@@ -847,26 +847,32 @@ def set(
     def subjects(
         self,
         predicate: Union[None, Path, _PredicateType] = None,
-        object: Optional[_ObjectType] = None,
+        object: Optional[Union[_ObjectType, List[_ObjectType]]] = None,
```

Comment:
@nicholascar : Was there a reason this used `List[...]` instead of `Sequence[...]`?

Reply:
The use case I have in mind: I often load `set`s with `URIRef` instead of `list`s to de-duplicate.